Create ui-c8y-1022-4-0-device-tab-parameters-list.md (#3483)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1022-4-0-device-tab-parameters-list.md

---
date: ""

> title: Device tab parameters list.

Reviewer: ```suggestion
title: New device tab listing all device parameters
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4666
version: 1022.4.0
---

> To provide a better overview of the device parameters, a new **Parameters** tab has been added to the Device Management application. This tab lists all parameters reported by the device in a table format, including the parameter name, value, type, and timestamp of the last update. With this change, users can now easily view and monitor device parameters without the need to navigate through measurements or the complete device data. The parameters list is automatically updated whenever new parameters are reported by the device.

Reviewer: ```suggestion
To provide a better overview of the device parameters, a new **Parameters** tab has been added to the device details. This tab lists all parameters reported by the device in a table format, including the parameter name, value, type, and timestamp of the last update. With this change, users can now easily view and monitor device parameters without the need to navigate through measurements or the complete device data. The parameters list is automatically updated whenever new parameters are reported by the device.
```

> To provide a better overview of the device parameters, a new **Parameters** tab has been added to the device details. This tab lists all parameters reported by the device in a table format, including the parameter name, value, type, and timestamp of the last update. With this change, users can now easily view and monitor device parameters without the need to navigate through measurements or the complete device data. The parameters list is automatically updated whenever new parameters are reported by the device.

Reviewer 2: ```suggestion
To provide an overview of the device parameters, a new **Parameters** tab has been added to the device details. This tab lists all parameters reported by the device in a table format, including the parameter title, key, value and type.
```

> To provide a better overview of the device parameters, a new "Parameters" tab has been added to the Device Management application. This tab lists all the parameters reported by the device in a table format, including the parameter name, value, type, and timestamp of the last update. With this change, users can now easily view and monitor the parameters of a device without needing to navigate through the measurements or the complete device data. The parameters list is automatically updated whenever new parameters are reported by the device.

Reviewer: ```suggestion
To provide a better overview of the device parameters, a new **Parameters** tab has been added to the Device Management application. This tab lists all parameters reported by the device in a table format, including the parameter name, value, type, and timestamp of the last update. With this change, users can now easily view and monitor device parameters without the need to navigate through measurements or the complete device data. The parameters list is automatically updated whenever new parameters are reported by the device.
```
